src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -205,10 +205,14 @@ def update_app_settings(cmd, resource_group_name, name, settings=None, slot=None
         for s in src:
             try:
                 temp = shell_safe_json_parse(s)
+                print(temp)
                 if isinstance(temp, list):  # a bit messy, but we'd like accept the output of the "list" command
                     for t in temp:
+                        print(t)
                         if t.get('slotSetting', True):
+                            print("Running with slot settings")
```

Comment:
Too many print(). Use logger?

Reply:
Sorry that was not meant to be here. I will remove.